test/bundle-size.test.ts

```diff
@@ -0,0 +1,16 @@
+import {expect, test} from 'vitest';
```

Comment:
Aren't these utilities already defined globally for every test? Or haven't we configured that in `carto-api-client`?

Reply:
Didn't realize that was [a built-in option](https://vitest.dev/config/#globals) in Vitest! I'd just thought of it as being a Jest thing. Personally it's a little too much magic for me (TS auto-imports are near-instant in these smaller repositories anyway) but I'd be OK with adding it if that seems like a direction we want to go for consistency.